packages/core-svelte/src/lib/components/TextComponent.svelte

```diff
@@ -710,10 +709,10 @@
                 />
                 <span class="text-component-width" bind:this={widthSpan}></span>
             </span>
-        {:else}
+            {:else}
             <!-- contenteditable must be true, otherwise there is no cursor position in the span after a click,
-				 But ... this is only a problem when this component is inside a draggable element (like List or table)
-			-->
+                 But ... this is only a problem when this component is inside a draggable element (like List or table)
+            -->
```

Comment:
Indentation of {:else} is now incorrect, it was correct, please change this back to what it was.